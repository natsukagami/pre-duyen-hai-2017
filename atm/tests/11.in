245 168
-6237751164792129122229255896512086483861746044128407431053889949291544955083437500408853542810169801177179320539573607725821709908388328985214304658306692154427894442307229651870126173011333803879458899610007360616740675202812072406464678710596
