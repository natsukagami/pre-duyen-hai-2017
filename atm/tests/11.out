99999999999999999987733803879458899610007360616740675202812072406464678710596
